### PR TITLE
Skip job if package is not on CRAN

### PR DESCRIPTION
**Pull request by m7pr**
Skips the job if the package is not on CRAN. Related to packages like `teal.osprey` or `teal.goshawk`

### CODE REVIEW

**Comment by m7pr** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by m7pr** [comment]
recheck

**Comment by pawelru** [comment]
Thanks for the PR however I would like to not to resign from testing non-CRAN packages. We do have quite a few of them and they are relevant for our internal process. Please give me some time on this. I might have an idea for this

**Comment by m7pr** [comment]
We can change the installation strategy to devtools::install_local if cran installation is not possible for the tested package

File: script.R [hunk 1]
```diff
@@ -65,7 +65,17 @@ cli::cli_progress_bar()
 # cache pkg and its dependencies
 cli::cli_progress_step("Installing the package...")
 pkg_name <- read.dcf("DESCRIPTION")[1, "Package"][[1]]
-crancache::install_packages(pkg_name, quiet = TRUE)
+tryCatch(
+  crancache::install_packages(pkg_name, quiet = TRUE),
+  warning = function(w) {
+    if (grepl('package.*is not available for this version of R', w$message)) {
```
Comment (cicdguy) [inline script.R]:
```suggestion
    if (grepl("package.*is not available for this version of R", w$message)) {
```

Fix linter warning.

**Comment by pawelru** [comment]
closed in favour of https://github.com/insightsengineering/r-revdepcheck-action/pull/4

**Pull request closed without merging**